src/app/Library/CrudPanel/Traits/FieldsProtectedMethods.php

```diff
@@ -144,6 +145,10 @@ protected function makeSureFieldHasEntity($field)
             // if it has parameters it's not a relation method.
             $field['entity'] = $this->modelMethodHasParameters($model, $possibleMethodName) ? false : $field['name'];
 
+            if ($field['entity']) {
+                $field['nestedEntity'] = $possibleMethodName;
```

Comment:
This is something I will probably remove tomorrow as I think I endup not needing it.